docs/03-tutorials/00-api-exposure/apix-09-expose-and-secure-workload-with-certificate.md

```diff
@@ -0,0 +1,291 @@
+---
+title: Expose and secure a workload with a certificate 
+---
+
+This tutorial shows how to expose and secure a workload with mutual authentication using Kyma's mutual TLS Gateway. 
+
+## Prerequisites
+
+This tutorial is based on a sample HttpBin service deployment and a sample Function. To deploy or create one of those, follow the [Create a workload](./apix-01-create-workload.md) tutorial.
+
+Before you start, Set up [`kyma-mtls-gateway`](../00-security/sec-02-setup-mtls-gateway.md) to allow mutual authentication in Kyma and make sure that you exported the [bundle certificates](../00-security/sec-02-setup-mtls-gateway#steps). 
+
+## Expose and access your workload
+
+Follow the instruction to expose and access your instance of the HttpBin service or your sample Function.
+
+<div tabs>
+  <details>
+  <summary>
+  HttpBin
+  </summary>
+
+1. Export the following values as environment variables:
+
+     ```bash
+   export DOMAIN_TO_EXPOSE_WORKLOADS={DOMAIN_NAME}
+   export GATEWAY=$NAMESPACE/$MTLS_GATEWAY_NAME
+   export CLIENT_CERT_CRT_FILE=client.example.com.crt
+	 export CLIENT_CERT_CSR_FILE=client.example.com.csr
+	 export CLIENT_CERT_KEY_FILE=client.example.com.key 
+   ```
+   >**NOTE:** `DOMAIN_NAME` is the domain that you own, for example, api.mydomain.com
+
+2. Generate Client certificate signed by Client Root CA: 
+
+   ```bash
+   # Create a new key and CSR for the client certificate
+   openssl req -out ${CLIENT_CERT_CSR_FILE} -newkey rsa:2048 -nodes -keyout ${CLIENT_CERT_KEY_FILE} -subj "/CN=client.example.com/O=example"
+   # Sign the client cert with CA cert
+   openssl x509 -req -days 365 -CA ${CLIENT_ROOT_CA_CRT_FILE} -CAkey ${CLIENT_ROOT_CA_KEY_FILE} -set_serial 0 -in ${CLIENT_CERT_CSR_FILE} -out ${CLIENT_CERT_CRT_FILE}
+   ```
+  
+  </details>
+  <details>
+  <summary>
+  Function
+  </summary>
+  
+1. Export the following values as environment variables:
+
+     ```bash
+   export DOMAIN_TO_EXPOSE_WORKLOADS={DOMAIN_NAME}
+   export GATEWAY=$NAMESPACE/$MTLS_GATEWAY_NAME
+   export CLIENT_CERT_CRT_FILE=client.example.com.crt
+	 export CLIENT_CERT_CSR_FILE=client.example.com.csr
+	 export CLIENT_CERT_KEY_FILE=client.example.com.key 
+   ```
+   >**NOTE:** `DOMAIN_NAME` is the domain that you own, for example, api.mydomain.com
+
+2. Generate Client certificate signed by Client Root CA: 
+
+   ```bash
+   # Create a new key and CSR for the client certificate
+   openssl req -out ${CLIENT_CERT_CSR_FILE} -newkey rsa:2048 -nodes -keyout ${CLIENT_CERT_KEY_FILE} -subj "/CN=client.example.com/O=example"
+   # Sign the client cert with CA cert
+   openssl x509 -req -days 365 -CA ${CLIENT_ROOT_CA_CRT_FILE} -CAkey ${CLIENT_ROOT_CA_KEY_FILE} -set_serial 0 -in ${CLIENT_CERT_CSR_FILE} -out ${CLIENT_CERT_CRT_FILE}
+   ```
+
+  </details>
+</div>
+
+## Access the secured resources
```

Comment:
so far we have no VS created, workload is not accessible. as far as i understand this was done to showcase that workload is not secured. i would drop this section completely